### PR TITLE
Pass language

### PR DESCRIPTION
**Pull request by mturnwall**
You can now pass a language in to override the browser's preferred language.

I also added the example page.

### CODE REVIEW

File: localization-bar.js [hunk 1]
```diff
@@ -25,6 +36,7 @@ export default class {
    */
   check() {
     this.language = this.findLanguage();
+    console.log(this.language);
```
Comment (jgallen23) [inline localization-bar.js]: Oops

File: example/example.js [hunk 2]
```diff
@@ -0,0 +1,237 @@
+/******/ (function(modules) { // webpackBootstrap
```
Comment (jgallen23) [inline example/example.js]: We shouldn't check this in

File: package.json [hunk 3]
```diff
@@ -5,7 +5,9 @@
   "main": "dist/localization-bar.js",
   "scripts": {
     "start": "scriptkit dev",
-    "prepublish": "scriptkit"
+    "prepublish": "scriptkit",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
```
Comment (jgallen23) [inline package.json]: Scriptkit already does the building, not sure why we added webpack

**Review by jgallen23 — CHANGES_REQUESTED** [review]

**Comment by mturnwall** [comment]
I removed the console log and the webpack config.

**Pull request merged**